src/renderer/theme.store.ts

```diff
@@ -113,25 +119,12 @@ export class ThemeStore extends Singleton {
     });
   }
 
-  bindNativeThemeUpdateEvent() {
-    ipcRenderer.on(setNativeThemeChannel, (event, theme: "dark" | "light") => {
-      this.osNativeTheme = theme;
-      this.applyTheme(theme);
```

Comment:
This call was the main source of the crash because it treated `"dark"` as a `ThemeId`